seed/views/v3/labels.py

```diff
@@ -0,0 +1,121 @@
+# !/usr/bin/env python
+# encoding: utf-8
+"""
+:copyright (c) 2014 - 2020, The Regents of the University of California, through Lawrence Berkeley National Laboratory (subject to receipt of any required approvals from the U.S. Department of Energy) and contributors. All rights reserved.  # NOQA
+:author 'Piper Merriam <pmerriam@quickleft.com>'
+"""
+from drf_yasg.utils import swagger_auto_schema
+from rest_framework.parsers import JSONParser, FormParser
+from rest_framework.renderers import JSONRenderer
+
+from seed.decorators import DecoratorMixin
+from seed.filters import (
+    LabelFilterBackend,
+)
+from seed.models import (
+    StatusLabel as Label,
+)
+from seed.serializers.labels import (
+    LabelSerializer,
+)
+from seed.utils.api import drf_api_endpoint
+from django.utils.decorators import method_decorator
+from seed.utils.api_schema import AutoSchemaHelper
+from seed.utils.labels import filter_labels_for_inv_type
+from seed.utils.viewsets import SEEDOrgNoPatchOrOrgCreateModelViewSet
+
+
+@method_decorator(
+    name='retrieve',
+    decorator=swagger_auto_schema(
+        manual_parameters=[AutoSchemaHelper.query_org_id_field(required=False, description="Optional org id which "
+                                                                                           "overrides the users "
+                                                                                           "(default) current org id")]
```

Comment:
Can we change the format of these? It makes it visually more difficult to pick up on what's going on to see the description broken up like this. (These method_decorators are already visually hard to track - we don't want to add to make it worse.)

Maybe have `required` and `description` be on their own lines?
```
        manual_parameters=[AutoSchemaHelper.query_org_id_field(
            required=False,
            description="Optional org id which overrides the users (default) current org id"
        )]
```